src/main/java/org/jenkinsci/plugins/edithelp/EditHelpRootAction.java

```diff
@@ -0,0 +1,148 @@
+package org.jenkinsci.plugins.edithelp;
+
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.BufferedWriter;
+import java.io.OutputStreamWriter;
+import java.io.FileOutputStream;
+import hudson.Extension;
+import hudson.cli.PrivateKeyProvider;
+import hudson.model.RootAction;
+import jenkins.model.Jenkins;
+import org.apache.commons.lang.StringUtils;
+import org.kohsuke.stapler.Stapler;
+
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import java.util.logging.Logger;
+
+import static java.nio.file.Files.readAllLines;
+import static java.util.logging.Level.FINE;
+
+
+@Extension
+public class EditHelpRootAction implements RootAction {
+
+    private static final Logger LOGGER = Logger.getLogger(PrivateKeyProvider.class.getName());
+    //reading the catalog of helpmanager
+    private static ArrayList<File> listWithFileNames = new ArrayList<>();
+    public static void doListFiles(String str) {
```

Comment:
if its a web method - it should not actually list files as of it can be vulnerability. If not - it should not start with `do`